owners/package.json

```diff
@@ -24,14 +24,14 @@
     "lodash": "4.17.15",
     "minimatch": "3.0.4",
     "probot": "7.5.3",
-    "sleep-promise": "8.0.1"
+    "sleep-promise": "8.0.1",
+    "smee-client": "1.1.0"
```

Comment:
This should only be used as a dev dependency. GAE exposes externally accessible URLs for the webhooks and those should be used by the github app

Reply:
Oh, my understanding was the opposite. I'll revert that then

Reply:
Do you know what the default webhook route would be?